packages/pigeon/lib/generator_tools.dart

```diff
@@ -116,3 +116,7 @@ HostDatatype getHostDatatype(
     return HostDatatype(datatype: datatype, isBuiltin: true);
   }
 }
+
+/// Warning printed at the top of all generated code.
+const String generatedCodeWarning =
+    'Autogenerated from pigeon, do not edit directly.';
```

Comment:
Capitalize Pigeon and add a context link for readers

Reply:
Done.